sale_showorder_ipf_client/models/client_config.py

```diff
@@ -174,15 +174,16 @@ def post_order(self):
                                      params=querystring)
         print(response.text)
 
-    def get_order_id(self):
+    def get_order_id(self,order_id):
+        # ~ 'preleminär','definitiv','avbruten','avbruten levererad',BÄR_status
```

Comment:
Is this different states of the order, why are they here if the function only gets the order id...?

Reply:
Its a reminder for the new module outplacement_order